testing/functional/tests/scheduler/resizeAppointments/timeline.ts

```diff
@@ -58,4 +62,31 @@ test(`Resize in the "timelineMonth" view`, async t => {
         .expect(resizableAppointment.date.startTime).eql(`10:00 AM`)
         .expect(resizableAppointment.date.endTime).eql(`11:00 AM`);
 
-}).before(() => createScheduler('timelineMonth', dataSource));
+}).before(() => createScheduler({
+    views: ['timelineMonth'],
```

Comment:
What do you think about removing this **'views: [view]'** argument?

Reply:
See above.